### PR TITLE
Tweak aste tutorial documentation

### PR DESCRIPTION
**Pull request by davidscn**
Follow-up to #244

### CODE REVIEW

**Review by MakisH — COMMENTED** [review]
Why this change?

**Comment by davidscn** [comment]
https://github.com/precice/tutorials/actions/runs/3082709094/jobs/4982738464

File: tools/clean-tutorial-base.sh [hunk 1]
```diff
@@ -7,5 +7,5 @@ set -e -u
 clean_tutorial .
 clean_aste 
 clean_precice_logs .
-rm -fv *.log
+rm -fv -- *.log
```
Comment (MakisH) [inline tools/clean-tutorial-base.sh:10]: For consistency with other places where we do a similar operation, I would rather do:

```suggestion
rm -fv ./*.log
```

Interestingly, this is not being triggered on my system, with shellcheck 0.8.0.

**Comment by davidscn** [comment]
Might be useful to first merge the proposed changes here and render the content on the website. Please merge in case you think this makes more sense.

**Review by MakisH — APPROVED** [review]
A couple of tiny language edits (very good already!).

The tutorial runs for me. :+1:

File: aste-turbine/run.sh [hunk 2]
```diff
@@ -6,29 +6,31 @@ set -e -x
...
-# Map from the finer mesh to coarser mesh
-mpirun -n 2 preciceMap -v -p A --mesh fine_mesh/fine_mesh --data "EggHolder" &
-mpirun -n 2 preciceMap -v -p B --mesh coarse_mesh/coarse_mesh --output mapped/mapped --data "InterpolatedData"
+# Map from fine mesh to coarse mesh, start to ASTE instances, one for each participant
+mpirun -n 2 precice-aste-run -p A --mesh fine_mesh/fine_mesh --data "Franke" &
+mpirun -n 2 precice-aste-run -p B --mesh coarse_mesh/coarse_mesh --output mapped/mapped --data "InterpolatedData"
 
-# Join the output files together to result.vtk,
+# Join the output files together to result.vtu
 # Recovery cannot be used since GlobalID's are not exist in mapped mesh
```
Comment (uekerman) [inline aste-turbine/run.sh:31]: I don't understand this last sentence. `--recovery` is used below ??

File: aste-turbine/README.md [hunk 3]
```diff
@@ -1,29 +1,63 @@
...
-* Usage of ASTE partitioner.
-* Usage of ASTE mesh joiner.
-* Usage of ASTE calculator for calculating a function on a given mesh.
-* Usage of ASTE.
+{% note %}
+The error measures used here are only useful for consistent mapping configurations, i.e., `constraint="consistent"`.
+{% endnote %}
 
-## Running ASTE
+This tutorial is meant as a starting point to investigate mapping setups. The provided configuration uses a `nearest-neighbor` mapping, but there are other mapping configurations available (commented out) in the `precice-config.xml` file. Example: using the last configuration (`rbf-compact-polynomial-c6` with a dense matrix decomposition) leads to the following error measures:
```
Comment (uekerman) [inline aste-turbine/README.md:48]: This variant takes too long, I killed it after 20 mins on my laptop. Can we choose sth different?

**Review by uekerman — APPROVED** [review]
I tweaked the language a bit. Two comments below, otherwise good to go. Please merge directly when done.

File: aste-turbine/README.md (hunk 3 above)
Comment (davidscn) [reply aste-turbine/README.md:48]: I was already a bit confused as quite some settings failed. Using preCICE in release, it went through in a minute, maybe two. I have to investigate..

Comment (uekerman) [reply aste-turbine/README.md:48]: Yes, I did use preCICE in Debug mode.

Comment (davidscn) [reply aste-turbine/README.md:48]: I swapped now coarse and fine.

**Pull request merged**